package.json

```diff
@@ -111,7 +111,7 @@
     "classnames": "^2.3.2",
     "framer-motion": "^6.5.1",
     "lodash-es": "^4.17.21",
-    "monday-ui-style": "0.1.206",
+    "monday-ui-style": "0.1.206-ui-background-color-91321e78d7.0",
```

Comment:
Note to self: replace with actual release

Reply:
you want to merge it with this version?

Reply:
ok cool